pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -222,9 +222,9 @@ public static void validateInstancePoolsNReplicaGroups(TableConfig tableConfig)
   }
 
   private static Set<ValidationType> parseTypesToSkipString(@Nullable String typesToSkip) {
-    return typesToSkip == null ? Collections.emptySet()
-        : Arrays.stream(typesToSkip.split(",")).map(s -> ValidationType.valueOf(s.toUpperCase()))
-            .collect(Collectors.toSet());
+    return typesToSkip == null ? Collections.emptySet() : Arrays.stream(typesToSkip.split(","))
+        .map(s -> ValidationType.valueOf(s.toUpperCase()))
```

Comment:
why format has been changed for these lines?

Reply:
it's changed by auto reformat, has already rollback these irrelevant cahnges